sechub-commons-model/src/main/java/com/mercedesbenz/sechub/commons/model/SecHubConfigurationModelValidator.java

```diff
@@ -411,8 +420,18 @@ private void validateHeaderNamesAndValues(WebScanConfigurationModelValidationCon
                 return;
             }
             if (sanatizedHttpHeader.getValue() == null || sanatizedHttpHeader.getValue().isEmpty()) {
-                webScanContext.markAsFailed(WEB_SCAN_NO_HEADER_VALUE_DEFINED, "The header name is : " + sanatizedHttpHeader.getName());
-                return;
+                Set<String> use = sanatizedHttpHeader.getNamesOfUsedDataConfigurationObjects();
```

Comment:
It would be nice to have some little comment here.

E.g.: 

```suggestion
                /* a http header config must have a value or a usage defined - here we have no value so a usage must be defined! */  
                Set<String> use = sanatizedHttpHeader.getNamesOfUsedDataConfigurationObjects();
```